packages/block-library/src/video/edit.native.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { View, TextInput, TouchableWithoutFeedback, Text } from 'react-native';
+/**
+ * Internal dependencies
+ */
+import Video from './video-player';
+import {
+	requestMediaImport,
+	mediaUploadSync,
+	requestImageFailedRetryDialog,
+	requestImageUploadCancelDialog,
+} from 'react-native-gutenberg-bridge';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Toolbar,
+	ToolbarButton,
+	Dashicon,
+} from '@wordpress/components';
+import {
+	MediaPlaceholder,
+	MediaUpload,
+	MEDIA_TYPE_VIDEO,
+	RichText,
+	BlockControls,
+	InspectorControls,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import { isURL } from '@wordpress/url';
+import { doAction, hasAction } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import styles from '../image/styles.scss';
+import MediaUploadProgress from '../image/media-upload-progress';
+import style from './style.scss';
+
+const VIDEO_ASPECT_RATIO = 1.7;
+
+class VideoEdit extends React.Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			showSettings: false,
+			isMediaRequested: false,
+			videoContainerHeight: 0,
+		};
+
+		this.mediaUploadStateReset = this.mediaUploadStateReset.bind( this );
+		this.onSelectMediaUploadOption = this.onSelectMediaUploadOption.bind( this );
+		this.finishMediaUploadWithSuccess = this.finishMediaUploadWithSuccess.bind( this );
+		this.finishMediaUploadWithFailure = this.finishMediaUploadWithFailure.bind( this );
+		this.updateMediaProgress = this.updateMediaProgress.bind( this );
+		this.onVideoPressed = this.onVideoPressed.bind( this );
+		this.onVideoContanerLayout = this.onVideoContanerLayout.bind( this );
+	}
+
+	componentDidMount() {
+		const { attributes, setAttributes } = this.props;
+		if ( attributes.id && ! isURL( attributes.src ) ) {
+			if ( attributes.url.indexOf( 'file:' ) === 0 ) {
```

Comment:
Hey @pinarol @SergioEstevao do we need this block of code as it was put for copy/paste images on iOS but not sure for videos?

Reply:
It looks like we don't have support for video copy paste on Aztec either. So yes, we don't need this code, let me remove that.